source/features/next-prev-notifications.css

```diff
@@ -0,0 +1,3 @@
+.flex-items-between {
```

Comment:
This class already exists in GitHub’s own CSS utilities, maybe named something different, we don't need a CSS file here